src/java/com/libmailcore/HTMLRendererTemplateCallback.java

```diff
@@ -12,9 +12,9 @@
     /** Returns true if an attachment should (such as an image) be shown. */
     boolean shouldShowPart(AbstractPart part);
     /** Returns the values used in the header template for a given template. */
-    Map<String, String> templateValuesForHeader(MessageHeader header);
+    Map<String, Object> templateValuesForHeader(MessageHeader header);
```

Comment:
The default callback in src/core/renderer/MCHTMLRendererCallback.cpp returns empty adds empty HashMaps to the result.

result->setObjectForKey(MCSTR("NOFROM"), mailcore::HashMap::hashMap());

In order to mimic that behavior, it was necessary to return a Map<String, Object>.